### PR TITLE
Change code proposal to object as strings are not supported

### PR DESCRIPTION
**Pull request by jatgarg**
Refer: https://github.com/microsoft/FluidFramework/pull/6196

### CODE REVIEW

File: snapshotTestContent/Authoring-MVP3/current_snapshots/snapshot_9000_0.json [hunk 1]
```diff
@@ -1050,7 +1050,7 @@
         "path": "quorumValues",
         "type": "Blob",
         "value": {
-          "contents": "[[\"code2\",{\"approvalSequenceNumber\":3,\"commitSequenceNumber\":4,\"key\":\"code2\",\"sequenceNumber\":2,\"value\":\"@ms/office-fluid-container\"}],[\"leader\",{\"approvalSequenceNumber\":3837,\"commitSequenceNumber\":3838,\"key\":\"leader\",\"sequenceNumber\":3835,\"value\":\"c3931334-2f54-414d-932d-002cee6f4280\"}]]",
```
Comment (anthony-murphy) [inline snapshotTestContent/Authoring-MVP3/current_snapshots/snapshot_9000_0.json:1053]: can you also change code2 to code. rather get rid of that as nothing should be using it. it's very old

Comment (jatgarg) [reply snapshotTestContent/Authoring-MVP3/current_snapshots/snapshot_9000_0.json:1053]: You mean remove the first element of array which is "code2"?

Comment (jatgarg) [reply snapshotTestContent/Authoring-MVP3/current_snapshots/snapshot_9000_0.json:1053]: Isn't it the key-value pair?

Comment (jatgarg) [reply snapshotTestContent/Authoring-MVP3/current_snapshots/snapshot_9000_0.json:1053]: Would not the snapshot tests fails which matches the snapshots?

Comment (anthony-murphy) [reply snapshotTestContent/Authoring-MVP3/current_snapshots/snapshot_9000_0.json:1053]: i'm saying change all instance of `code2` to `code` just like you did by making the value and object

Comment (jatgarg) [reply snapshotTestContent/Authoring-MVP3/current_snapshots/snapshot_9000_0.json:1053]: Done. I got confused by when you said get rid of it. :)

**Review by anthony-murphy — APPROVED** [review]

**Pull request merged**